src/map/mob.c

```diff
@@ -2485,7 +2485,7 @@ int mob_dead(struct mob_data *md, struct block_list *src, int type)
 				drop_rate = max(drop_rate,cap_value((int)(0.5+drop_rate*(sd->sc.data[SC_ITEMBOOST]->val1)/100.),0,9000));
 			// Increase item drop rate for VIP.
 			if (battle_config.vip_drop_increase && (sd && pc_isvip(sd))) {
-				drop_rate += (int)(0.5 + (drop_rate * battle_config.vip_drop_increase) / 10000.);
+				drop_rate += (int)(0.5 + (drop_rate * battle_config.vip_drop_increase) / 100);
```

Comment:
On actual drop, we were adding
e.g. 1% drop_rate = 100, vip_drop_increase = 50, drop_rate += 0.5 + (100*50)/10000, which is 1, so the drop rate increased to 101.
Now:
1% drop rate = 100, vip_drop_increase = 50, drop_rate += 0.5 + (100*50)/100, we are adding 50 to drop rate and effectively increasing drop rate by 50% as expected.